gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-all.zip
+# downgrade to 4.8.1 due to Lombok issues with 4.9
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.8.1-all.zip
 zipStoreBase=GRADLE_USER_HOME
```

Comment:
Do we need to include files from your wrapper directory??

Reply:
thanks for the comment, run the command and made sure the jar file is updated and added a comment to README.md file for the future updates